packages/terra-functional-testing/src/services/wdio-terra-service.js

```diff
@@ -5,13 +5,38 @@ const { toBeAccessible } = require('../commands/expect');
 const { getViewports, describeViewports, setViewport } = require('../commands/viewport-helpers');
 
 const hideInputCaret = require('../commands/hide-input-caret');
+const { element } = require('../commands/validates');
 
 class TerraService {
   constructor(options = {}) {
-    const { formFactor, theme } = options;
+    this.serviceOptions = {
+      theme: 'terra-default-theme',
+      ...options,
+    };
+  }
+
+  onPrepare(config) {
+    // Add the configurations at the top level service options to the Terra Service options.
+    this.serviceOptions = {
+      selector: '[data-terra-test-content] *:first-child',
+      ...this.serviceOptions,
+      ...config.serviceOptions,
+    };
+  }
+
+  beforeSession() {
+    global.Terra = {
+      /*
+       This command must be defined in the beforeSession hook instead of together with the other Terra custom commands in the
+       before hook. The reason being webdriverio v6 now executed the describe block prior to running the before hook.
+       Therefore, this command needs to be defined before the test starts in the testing life cycle.
 
-    this.formFactor = formFactor;
-    this.theme = theme || 'terra-default-theme';
+       Reference: https://github.com/webdriverio/webdriverio/issues/6119
+      */
+      describeViewports,
+
+      serviceOptions: this.serviceOptions,
```

Comment:
To move this PR forward I think we should keep this as is, but in the long run I think we can/should avoid adding the options to the global namespace and call the function directly with the necessary parameters. It might end up looking something like this:

https://github.com/cerner/terra-toolkit/compare/bind-axe-options

In this case we bind the accessibility function to the instance and proxy the call with the configuration override. We could also bind the utility functions with the options we expect. 

```js
describeViewports: describeViewports.bind(null, formFactor); // Always call describeViewports with the form factor as the first parameter.
```